### PR TITLE
cmd: add pd-bench application

### PR DESCRIPTION
**Pull request by coocood**
Used to benchmark GetTS performance.

### CODE REVIEW

**Comment by coocood** [comment]
@huachaohuang @disksing PTAL

**Comment by huachaohuang** [comment]
LGTM

**Comment by siddontang** [comment]
I prefer rename to pd-tso-bench, Rest LGTM

File: cmd/pd-bench/main.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	s := newStats(*num)
+	for i := 0; i < s.count; i++ {
+		start := time.Now()
+		_, _, err := pdCli.GetTS()
+		if err != nil {
+			log.Fatal(err)
+		}
+		dur := time.Since(start)
+		s.update(dur)
+		time.Sleep(sleep)
```
Comment (siddontang) [inline cmd/pd-bench/main.go]: why sleep here?

Comment (coocood) [inline cmd/pd-bench/main.go]: To control the request pressure.

**Comment by coocood** [comment]
@siddontang PTAL

File: cmd/pd-tso-bench/main.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	"time"
+
+	"github.com/pingcap/pd/pd-client"
+)
+
+var (
+	pdAddrs     = flag.String("pd", "127.0.0.1:2379", "pd address")
+	concurrency = flag.Int("C", 100, "concurrency")
+	num         = flag.Int("N", 1000, "number of request per request worker")
+	sleepFlag   = flag.String("sleep", "1ms", "sleep time after a request, used to adjust pressure")
```
Comment (siddontang) [inline cmd/pd-tso-bench/main.go]: why not using flag.Duration directly?

**Comment by coocood** [comment]
@siddontang PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**